.idea/codeStyleSettings.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`.idea` files should probably be added in a separate commit, if at all. Ideally editor-specific formatting rules should be ignored by git (ie there should be a .gitignore rule for these files).